apis/eks/v1alpha1/types.go

```diff
@@ -51,7 +51,7 @@ type NodeGroupParameters struct {
 	//
 	// ClusterName is a required field
 	// +immutable
-	ClusterName *string `json:"clusterName,omitempty"`
```

Comment:
The EKS changes are not at risk of panic and do not actually change the generated CRD, but went ahead and fixed them while I was here.